apps/meteor/app/slashcommands-mute/server/mute.ts

```diff
@@ -11,44 +10,46 @@ import { api } from '../../../server/sdk/api';
  * Mute is a named function that will replace /mute commands
  */
 
-function Mute(_command: 'mute', params: string, item: IMessage): void {
-	const username = params.trim().replace('@', '');
-	if (username === '') {
-		return;
-	}
+slashCommands.add(
+	'mute',
+	function Mute(_command: 'mute', params, item): void {
```

Comment:
Nesting this function declaration as a parameter expression damages readability, as evidenced by the indentation increase.